openshift-templates/varnish-4-rhel7-ephemeral.yaml

```diff
@@ -0,0 +1,299 @@
+{
+  "kind": "Template",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "varnish-ephemeral",
+    "annotations": {
+      "openshift.io/display-name": "Varnish 4 (Ephemeral)",
+      "description": "Varnish HTTP Cache container images based on Red Hat Software Collections and intended for OpenShift and general usage.  For more information about using this template, including OpenShift considerations, see https://github.com/sclorg/varnish-container/blob/master/README.md.",
+      "tags": "quickstart,varnish",
+      "iconClass": "icon-other-unknown",
+      "template.openshift.io/long-description": "This container image includes Varnish 4.0 Cache server and a reverse proxy for OpenShift and general usage.",
```

Comment:
The version should not be here, as you can have multiple versions instantiated from one template (container-wise it's just a tag, so it can be [parametrized](https://github.com/sclorg/postgresql-container/blob/master/examples/postgresql-ephemeral-template.json#L246)).